static/css/tidyverse-site.css

```diff
@@ -912,7 +912,7 @@ pre, code {
 
 pre {
   padding: 10px;
-  line-height: 2em;
+  line-height: 1.5em;
```

Comment:
Should this just be `1.5`? I have a vague recollection that's safer.

Reply:
I think it has to have a unit unless it is `0` when (you are right) it is safest to omit it.

Reply:
It's not a regular unit: https://css-tricks.com/almanac/properties/l/line-height/

Reply:
Oh, interesting! TIL, I have changed it to be unitless